docs/HPCCSystemAdmin/SA-Mods/SecMgrModConfDeploy.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="configure-and-deploy-security-manager-plugin">
+  <title role="nbrk">Configure and Deploy the Security Manager Plug-in</title>
+
+  <para>The following sections detail the process of how to configure your
+  HPCC system to use the Security Manager Plug-in.</para>
```

Comment:
The following sections detail the process of configuring your HPCC system to use the Security Manager Plug-in.